src/main/java/com/bobocode/orm/action/InsertAction.java

```diff
@@ -0,0 +1,21 @@
+package com.bobocode.orm.action;
+
+import com.bobocode.orm.jdbc.JDBCRepository;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public class InsertAction implements EntityAction {
+
+  private final Object entity;
+  private final JDBCRepository jdbcRepository;
+
+  @Override
+  public void execute() {
+    jdbcRepository.insert(entity);
+  }
+
+  @Override
+  public int priority() {
+    return 1;
```

Comment:
I would consider to avoid magic numbers. Use constants instead. Maybe separate class with constants

Reply:
fixed